orders/parser.go

```diff
@@ -0,0 +1,65 @@
+package orders
+
+import (
+	"fmt"
+
+	dip "github.com/zond/godip/common"
+)
+
+type Parser struct {
+	generators []dip.Order
```

Comment:
Maybe call this something else? "prototypes" might work, but I'm not dogmatic :)